sdk/eventhub/Microsoft.Azure.EventHubs.Processor/src/AzureStorageCheckpointLeaseManager.cs

```diff
@@ -489,9 +491,6 @@ async Task<bool> UpdateLeaseCoreAsync(AzureBlobLease lease)
                 return false;
             }
 
-            // First, renew the lease to make sure the update will go through.
-            await this.RenewLeaseAsync(lease).ConfigureAwait(false);
```

Comment:
Lease renewal is responsibility of Partition Manager. If lease is lost, UploadText will fail anyways. There is no point to have a renew attempt here while PartitionManager is already doing its job.

Reply:
That's pretty much what I was thinking, looking at the change. I believe I only put the renew in Java because it was there in SJ's code for the original EPH.

Reply:
I think it is a better approach to delegate responsibility to single agent. This feels better for debugging as well especially when lease expires and you don't need to ask if customer checkpointed regularly.